src/drivers/i2c/i2c.h

```diff
@@ -0,0 +1,37 @@
+#ifndef I2C_H
+#define I2C_H
+
+#include "common.h" // Common definitions
+#include <stdint.h> // Standard integer types
+
+// Supported I2C devices
+typedef enum {
+    I2C_DEVICE_POLULU, // Polulu AltIMU-v6
+    I2C_DEVICE_MOVELLA, // Movella MTi-630 IMU
```

Comment:
the movella talks over UART, not I2C. Any references to it in I2C should be removed